src/chain.h

```diff
@@ -24,6 +24,7 @@ static const int SAPLING_VALUE_VERSION = 1010100;
 static const int CHAIN_HISTORY_ROOT_VERSION = 2010200;
 static const int NU5_DATA_VERSION = 4050000;
 static const int TRANSPARENT_VALUE_VERSION = 5040026;
+static const int NU6_DATA_VERSION = 5100025;
```

Comment:
A side-effect of using `5100025` here instead of `5100000` is that the data will not be read/written during the RC period (it will only start at the production 5.10.0 release). That's fine as long as we're not trying to test node restarts during the RC period (or we can alter this value specifically for the private testnet, and then make sure we delete the entire `zcashd` private testnet datastore afterwards to ensure it doesn't accidentally get used for public testnet).

Reply:
I think we should alter this value specifically for the private testnet.

Reply:
In this particular case, I don't see the harm in enabling this starting from the rc. It's only going to write zero for the new field.

Reply:
The reason that I would prefer we leave it as-is is that, since the field only becomes necessary as of NU6 (and, relatedly, zcash v6.0.0) it's fine if the value is never written by zcashd v5.10.0, and if we need to make any changes here in the RC period, we won't have to worry about people having potentially-corrupt state that we then have to deal with when making the v5.10.0 release.